plugins/attachment/src/index.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2021 Anticrm Platform Contributors.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Class, Doc, Ref } from '@anticrm/core'
+import type { Plugin, Service } from '@anticrm/platform'
+import { plugin } from '@anticrm/platform'
+import type { IntlString } from '@anticrm/status'
+
+export interface Attachment extends Doc {
+  objectId: Ref<Doc>
+  objectClass: Ref<Class<Doc>>
+  name: string
+  format: string
+  size: number
+}
+
+export function nameToFormat (name: string): string {
+  return name.slice(name.lastIndexOf('.') + 1)
```

Comment:
![изображение](https://user-images.githubusercontent.com/80476319/134101519-af663140-4e72-44fb-9e03-de411cab357b.png)
Used for this. Yes, we can store MIME and contert to file extension use some library, but why we need this?

Reply:
It is just more consistent (for instance, `.jpeg`/`.jpg`), but ideally mime type should be checked on backend side since the user can rename extension as he want.